lib/please_cli/please_cli/check.py

```diff
@@ -43,8 +43,27 @@
         please_cli.config.NIX_BIN_DIR + 'nix-shell',
         ),
     )
+@click.option(
```

Comment:
I would rather use `cli_common.click.taskcluster_options` decorator here, like in other commands. But I would definitely copy the default and help options you have here.

Reply:
I wasn't aware it existed, I'm using it now and since I was there I've also modified `run.py` and `shell.py` to use it.